### ISSUE TITLE
Create Postgres Adaptor

### ISSUE THREAD

**Issue by bastienboutonnet** [bastienboutonnet/sheetwork#256]
Here's an interesting article which may help when setting this up.
https://docs.github.com/en/free-pro-team@latest/actions/guides/creating-postgresql-service-containers

**Comment by bastienboutonnet** [comment]
Starting work on this.

TODO:
- [ ] try and get a sense of what db opearations are needed so that I can fixate the requirements for the `Profile` object. I might be able to figure this out from the kind of requirements `dbt` puts on postgres setups. Right now it looks like this:
```yaml
postgres_db:
    db_type: postgres
    user: sheetwok_user
    password: magical_password
    host: localhost
    database: sheetwork_test
    schema: sheetwork_test_schema
    guser: sheetwork_test@blahh.iam.gserviceaccount.com
```

**Comment by bastienboutonnet** [comment]
Implemented by #314
